README.md

```diff
@@ -0,0 +1,97 @@
+# near-lake-framework
+
+NEAR Lake Framework is a small library companion to [NEAR Lake](https://github.com/near/near-lake). It allows you to build
+your own indexer that subscribes to the stream of blocks from the NEAR Lake data source and create your own logic to process
+the NEAR Protocol data.
+
+## Example
+
+```rust
+use futures::StreamExt;
+use near_lake_framework::LakeConfig;
+
+#[tokio::main]
+async fn main() -> Result<(), tokio::io::Error> {
+    // create a NEAR Lake Framework config
+    let config = LakeConfig {
+        bucket: "near-lake-testnet".to_string(), // AWS S3 bucket name
+        region: "eu-central-1".to_string(), // AWS S3 bucket region
+        start_block_height: Some(82422587), // the latest block height we've got from explorer.near.org for testnet
+        tracked_shards: vec![0, 1, 2, 3], // track all 4 existing shards
+    };
+
+    // instantiate the NEAR Lake Framework Stream
+    let stream = near_lake_framework::streamer(config);
+
+    // read the stream events and pass them to a handler function with
+    // concurrency 1
+    let mut handlers = tokio_stream::wrappers::ReceiverStream::new(stream)
+        .map(|streamer_message| handle_streamer_message(streamer_message))
+        .buffer_unordered(1usize);
+
+    while let Some(_handle_message) = handlers.next().await {}
+
+    Ok(())
+}
+
+// The handler function to take the entire `StreamerMessage`
+// and print the block height and number of shards
+async fn handle_streamer_message(
+    streamer_message: near_lake_framework::near_indexer_primitives::StreamerMessage,
+) {
+    eprintln!(
+        "{} / shards {}",
+        streamer_message.block.header.height,
+        streamer_message.shards.len()
+    );
+}
+```
+
+## How to use
+
+### AWS S3 Credentials
+
+In order to be able to get objects from the AWS S3 bucket you need to provide the AWS credentials.
```

Comment:
Is there a direct link to the page where it is described how to get those credentials?